tools/python/boutiques/validator.py

```diff
@@ -26,10 +26,17 @@ def validate_descriptor(json_file, **kwargs):
         schema = simplejson.load(fhandle)
 
     # Load descriptor
-    with open(json_file) as fhandle:
+    json_object = kwargs.get('json_object')
```

Comment:
Check if it's a file (calling the dedicated function) instead of using an argument